composer.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "swiftmailer/swiftmailer",
+    "name": "friendsofsymfony1/swiftmailer",
     "type": "library",
-    "description": "Swiftmailer, free feature-rich PHP mailer",
+    "description": "Fork of unmaingained Swiftmailer, free feature-rich PHP mailer",
```

Comment:
spelling error - should be `unmaintained`

Reply:
Done